scripts/soak_test/soak_test_send_notification.py

```diff
@@ -0,0 +1,38 @@
+import os
```

Comment:
We didn't want this to sit in a subfolder of tests-stress?

Reply:
🤔 We could think of this as a very low level stress test... generally I've been using high priority templates for these soak tests and low priority ones for the larger stress tests so it would require either rewriting the code a bit or changing the .env file when switching between them. 🤷